Sources/Sentry/SentryHub.m

```diff
@@ -672,6 +672,14 @@ - (void)flush:(NSTimeInterval)timeout
     }
 }
 
+- (void)close
+{
+    SentryClient *client = _client;
+    if (client != nil) {
+        [client close];
+    }
```

Comment:
l:  I think we should stop doing this kind of check since objc have "message to nil" protection by default.
```suggestion
    [_client close];
```